app/assets/javascripts/mobile_tooltip.js

```diff
@@ -0,0 +1,41 @@
+(function () {
+  /*
+  **  Tooltip for the profile page on mobile devices
+  **  This script should ideally run on the profile page
+
+  **  Behaviour
+      ========================
+      - Enables the viewers tooltip to be shown on mobile devices when the
+        parent element is clicked.
+      - Toggle behaviour is implemented so that the tooltip can be both shown
+        and dismissed
+
+  */
+
+  $(document).on("page:load ready", function () {
+    // Allow this script to only be run on the profile page
+
+    if (window.location.pathname !== "/profile") return false;
+
+    var toggleCount = 0;
+
+    // Ensure the script is activated on mobile devices only
+    // Max-width: 768px
+
+    if ($(window).width() <= 768) {
+      $(".some_focus_text").on("click", function () {
+        var child = $(this).children(".yes_title").eq(0),
+            title = child.prop("title");
+
+        child.tooltip({
+          content: title,
+          position: { my: "top-10 left+25", at: "top left", of: $(this) }
+        });
+
+        toggleCount++;
+
+        toggleCount % 2 === 1 ? child.tooltip("open") : child.tooltip("close");
```

Comment:
Okay, I experimented with this, and this works most of the time, but consider this situation:

- There are 3 elements with the class `some_focus_text`
- I click on the first element -> toggleCount increases to 1 -> first tooltip shows
- I click on the second element -> toggleCount increases to 2 -> nothing happens, because toggleCount % 2 is 0, and it tries to close something that isn't there
- I click on the first element again -> toggleCount increases to 3 -> nothing happens, because toggleCount % 2 is 1, but the tooltip is already open so opening it again does nothing
- I click on the first element a third time -> toggle Count increases to 4 -> the first tooltip finally closes

What would be better is not tracking how many times we click any element with the class `some_focus_text` but rather ask the element being clicked on if it has a tooltip open or not already, and do the opposite action of that. After some poking around, I found that the `yes_title` span gets an `aria-describedby` attribute with the ID of a tooltip if that tooltip is already open. So you could do something like:

```
var tooltipId = $(child).attr('aria-describedby');

if (tooltipId) {
  child.tooltip("close");
} else {
  child.tooltip("open");
}
```

and then things open/close as expected even given the above scenario.